src/Features/Core/Portable/AddParameter/AbstractAddParameterCodeFixProvider.cs

```diff
@@ -69,15 +72,55 @@ internal abstract class AbstractAddParameterCodeFixProvider<
                 return null;
             }
 
+            if (this.CannotConvertDiagnosticIds.Contains(diagnostic.Id))
```

Comment:
do we always want this?  What if you ahve something like:

```c#
    static void Main(string[] args)
    {
        Foo(true, 4);
    }

    static void Foo(params int[] nums)
    {

    }
```

Won't we get "cannot convert bool to int".  but we'd still want to add a bool param.

Reply:
Added a unit test (TestInvocation_AddParameterToMethodWithParams)

Reply:
Can you add a doc comment to that effect?  thanks!